Modules/System/Azure Function/AzureFunctions.Codeunit.al

```diff
@@ -37,7 +48,7 @@ codeunit 7804 "Azure Functions"
     [NonDebuggable]
     procedure SendPostRequest(AzureFunctionsAuthentication: Interface "Azure Functions Authentication"; Body: Text; ContentTypeHeader: text): Codeunit "Azure Functions Response"
     begin
-        exit(AzureFunctionsImpl.SendPostRequest(AzureFunctionsAuthentication, Body, ContentTypeHeader));
+        exit(GetAzureFunctionInterface.SendPostRequest(AzureFunctionsAuthentication, Body, ContentTypeHeader));
```

Comment:
You must add parantheses to any procedure
>   Error: AA0008 You must specify open and close parenthesis after 'GetAzureFunctionInterface'.

Reply:
yeah, I saw it too, but I didn't have time. I fixed it now ;) Thx